packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -80,6 +82,21 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
       _width: (this._rootElem && this._rootElem!.offsetWidth)!,
       _height: (this._rootElem && this._rootElem!.offsetHeight - reducedHeight!)!,
     });
+
+    const isChartEmpty = !(
+      this.props.data &&
+      this.props.data.chartData &&
+      this.props.data.chartData!.filter((d: IChartDataPoint) => d.data! > 0).length
```

Comment:
be explicit that you are checking for length greater than 0.

Reply:
done at all places